examples/simpleapp/src/main/java/com/horizen/examples/SimpleApp.java

```diff
@@ -19,13 +23,33 @@ public static void main(String[] args) {
             System.out.println("File on path " + args[0] + " doesn't exist");
             return;
         }
-
         String settingsFileName = args[0];
 
+        // default name for the log file
+        String logFileName = "logs/debugLog.txt";
```

Comment:
Let's try to use the directory path specified in a configuration file: `scorex { logDir = "..." }`